Make email visibility default configurable in Slack import flow.

amanagr opened the pull request:
Fixes #34982


Tested by doing an import that email address visibility is set correct for realm default and in imported users.
Tested that the default email address visibility in the form is set based on org type.

<img width="573" height="474" alt="Screenshot from 2025-10-22 16-29-46" src="https://github.com/user-attachments/assets/1b9885b4-5a2f-4101-b01c-8e0c6e0d152a" />

alya commented on templates/zerver/slack_import.html:
```diff
@@ -82,6 +82,22 @@ <h1 class="get-started">{{ _("Import from Slack") }}</h1>
+                            <br />
+                            <span id="email-address-visibility-help-text">
+                                {% trans %}
+                                Users can <a href="https://zulip.com/help/configure-email-visibility" target="_blank">configure</a> email visibility for their own account.
```
```suggestion
                                Users can <a href="https://zulip.com/help/configure-email-visibility" target="_blank">change</a> their email visibility configuration when they log in.
```

amanagr commented:
> I think we can relabel the longest one as "Everyone" here?

Fixed


> For the other one, I think the page might feel better overall if we made the white box wider in any case, so we could widen that and this field enough to make it fit in English, at least. We have a lot of text that's going on two lines, which contributes to the height of the page, but I dunno if it'll feel too wide if we try to fix that.

Fixed this without widening the select element by reducing the empty space after the text.


<img width="573" height="474" alt="Screenshot from 2025-10-22 16-29-46" src="https://github.com/user-attachments/assets/da78b4d3-a787-4052-9b31-f9eb9846791d" />
